docs/docs/code/python/faqs/README.md

```diff
@@ -0,0 +1,25 @@
+# Frequently Asked Questions about Python
+
+## What's the difference between `def handler(pd)` and the `pipedream` package for Python code steps?
+
+The `handler` method is a newer implementation of Python code steps. It includes the ability to stop the workflow early, integrate a Data Store and/or Connected Accounts into your Python code step.
+
+However, at this time there are issues with our Python interpreter that is causing an `ECONNRESET` error.
```

Comment:
What about a **Known Issues** section? I'd like to list a couple of other things as we discover more bugs with Biz's new code.

Reply:
@ctrlaltdylan I'd recommend addressing this (especially since the `ECONNRESET` issue is happening less now).